inference-engine/ie_bridges/java/tests/BlobTests.java

```diff
@@ -13,6 +15,7 @@ public void testGetBlob(){
         Assert.assertArrayEquals(blob.getTensorDesc().getDims(), dimsArr);
     }
 
+    @Test
     public void testGetBlobFromFloat(){
```

Comment:
Missing space before bracket (for both tests).

Reply:
I think that the code style checker can solve such issues :)